pyproject.toml

```diff
@@ -0,0 +1,549 @@
th-callable", "nan-comparison", "syntax-error", "unrecognized-inline-option", "bad-plugin-value", "bad-configuration-section", "unrecognized-option", "logging-unsupported-format", "logging-format-truncated", "logging-too-many-args", "logging-too-few-args", "no-member", "not-callable", "assignment-from-no-return", "no-value-for-parameter", "too-many-function-args", "unexpected-keyword-arg", "redundant-keyword-arg", "missing-kwoa", "invalid-sequence-index", "invalid-slice-index", "assignment-from-none", "not-context-manager", "invalid-unary-operand-type", "unsupported-binary-operation", "repeated-keyword", "unsupported-membership-test", "unsubscriptable-object", "unsupported-assignment-operation", "unsupported-delete-operation", "invalid-metaclass", "dict-iter-missing-items", "await-outside-async", "unhashable-member", "not-an-iterable", "not-a-mapping", "invalid-unicode-codec", "bidirectional-unicode", "invalid-character-backspace", "invalid-character-carriage-return", "invalid-character-sub", "invalid-character-esc", "invalid-character-nul", "invalid-character-zero-width-space", "modified-iterating-dict", "modified-iterating-set", "method-hidden", "access-member-before-definition", "no-method-argument", "no-self-argument", "invalid-slots-object", "assigning-non-slot", "invalid-slots", "inherit-non-class", "inconsistent-mro", "duplicate-bases", "class-variable-slots-conflict", "invalid-class-object", "invalid-enum-extension", "non-iterator-returned", "unexpected-special-method-signature", "invalid-length-returned", "invalid-bool-returned", "invalid-index-returned", "invalid-repr-returned", "invalid-str-returned", "invalid-bytes-returned", "invalid-hash-returned", "invalid-length-hint-returned", "invalid-format-returned", "invalid-getnewargs-returned", "invalid-getnewargs-ex-returned", "undefined-variable", "invalid-all-format", "no-name-in-module", "potential-index-error", "relative-beyond-top-level", "bad-super-call", "bad-except-order", "raising-bad-type", "misplaced-bare-raise", "bad-exception-cause", "raising-non-exception", "notimplemented-raised", "catching-non-exception", "invalid-envvar-value", "yield-inside-async-function", "not-async-context-manager", "bad-format-character", "truncated-format-string", "mixed-format-string", "format-needs-mapping", "missing-format-string-key", "too-many-format-args", "too-few-format-args", "bad-string-format-type", "bad-str-strip-call", "init-is-generator", "return-in-init", "function-redefined", "not-in-loop", "return-outside-function", "yield-outside-function", "return-arg-in-generator", "nonexistent-operator", "duplicate-argument-name", "abstract-class-instantiated", "too-many-star-expressions", "invalid-star-assignment-target", "star-needs-assignment-target", "nonlocal-and-global", "continue-in-finally", "nonlocal-without-binding", "used-prior-global-declaration", "bad-reversed-sequence", "misplaced-format-function", "fatal", "astroid-error", "parse-error", "config-parse-error", "method-check-failed"]
```

Comment:
This looks to verbose. The options that work generally fine across many of projects I'm involved in are here
https://github.com/pymc-devs/pymc/blob/main/.pylintrc
The idea is to blacklist all and enable some of the checks, it should look more concise

Reply:
Addressed in [254124c](https://github.com/geoopt/geoopt/pull/201/commits/254124c325aa2e672e86327804c077bb4e7f8286)